apis/r/R/TileDBArray.R

```diff
@@ -161,6 +161,7 @@ TileDBArray <- R6::R6Class(
         isTRUE(simplify) || isFALSE(simplify),
         isTRUE(index1) || isFALSE(index1)
       )
+      .Deprecated(new="shape", msg="used_shape will be removed in TileDB-SOMA 1.14")
```

Comment:
```suggestion
      .Deprecated(new="shape", msg="The 'used_shape' function will be removed in TileDB-SOMA 1.14.")
```

Reply:
Full sentences are capitalization are a nice touch (but not a functional requirement).